### ISSUE TITLE
Namespaced model classes

### ISSUE THREAD

**Issue by tcannonfodder** [kaspth/oaken#69]
👋  What's the convention for accessing model classes that are namespaces?

For example; I have `Administrator` and `Administrator::Passkey` as 2 ActiveRecord models.

```ruby
(rdbg) administrators
#<struct Oaken::Stored::ActiveRecord
 type=
  Administrator(id: integer, email: string, webauthn_id: string, sign_in_count: integer, current_sign_in_at: datetime, last_sign_in_at: datetime, current_sign_in_ip: string, last_sign_in_ip: string, created_at: datetime, updated_at: datetime),
 key="administrators">
(ruby) administrator_passkeys
eval error: undefined local variable or method `administrator_passkeys' for Oaken::Seeds:Module
  /Users/tcannon/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/oaken-0.5.0/lib/oaken/seeds.rb:14:in `method_missing'
  (rdbg)/db/seeds/administrators/default.rb:1:in `block in load_onto'
nil
```

Thanks!

**Comment by elvinaspredkelis** [comment]
Super interested in this as well!

**Comment by kaspth** [comment]
Ah yeah, I just told @tcannonfodder privately and forgot to reply here.

Right now, you have to call `register Administrator::Passkey` because we don't auto-detect namespaces.

So we could use documentation for this, until I get to the auto-detection.

**Comment by kaspth** [comment]
Oh and I recommend calling `register` in `Oaken.prepare` in db/seeds.rb.

**Comment by kaspth** [comment]
Just auto-fixed this in #83.
